packages/editor-ui/src/components/WorkflowActivator.vue

```diff
@@ -192,7 +181,8 @@ export default mixins(
 	);
 </script>
 
-<style scoped>
+<style lang="scss" scoped>
```

Comment:
If need, I can update this with module one except for the last ::v-deep
Also there is class called could-not-be-started with margin-left: 0.5em; Maybe we can replace it with --spacing-2xs

Reply:
ya this is an old component i don't want to refactor right now 😲

Reply:
ideally we should not have to use v-deep